dist/table-dragger.js

```diff
@@ -72,8 +72,6 @@ return /******/ (function(modules) { // webpackBootstrap
 	  return _drag2.default.create(el, options);
 	};
 	exports.default = create;
-	
-	module.exports = create;
```

Comment:
I had to add this line because without this it wouldn't work

Reply:
ok, you tested it?